index.js

```diff
@@ -19,3 +20,15 @@ app.use('/', routes)
 app.listen(port, function () {
   console.log('app working on port:', port)
 })
+
+// If that above routes didnt work, we 404 them and forward to error handler
+app.use(errorHandlers.notFound)
+app.use(errorHandlers.catchErrors)
+// Otherwise this was a really bad error we didn't expect! Shoot eh
+if (app.get('env') === 'development') {
+  /* Development Error Handler - Prints stack trace */
+  app.use(errorHandlers.developmentErrors)
+}
+
+// production error handler
+app.use(errorHandlers.productionErrors)
```

Comment:
this "error handler" block should be before app.listen and after router middlware